base/loading.jl

```diff
@@ -1020,7 +1020,7 @@ function _include_from_serialized(pkg::PkgId, path::String, ocachepath::Union{No
                 elapsed = round((time_ns() - t_before) / 1e6, digits = 1)
                 comp_time, recomp_time = cumulative_compile_time_ns() .- t_comp_before
                 print(lpad(elapsed, 9), " ms  ")
-                for extid in EXT_DORMITORY
+                for extid in values(EXT_DORMITORY)
                     if extid.id == pkg
                         print(extid.parentid.name, " → ")
                         break
```

Comment:
This will not return true for any elemen, so it could be simply replaced here by:
```suggestion
                 if false; if false
```

Reply:
Yeah :(. This broke the time_imports pretty printing for extensions.